net-misc/bird/bird-2.16.2.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools fcaps
+
+DESCRIPTION="A routing daemon implementing OSPF, RIPv2 & BGP for IPv4 & IPv6"
+HOMEPAGE="https://bird.network.cz"
+SRC_URI="https://bird.network.cz/download/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~loong ~x86 ~x64-macos"
+IUSE="+client custom-cflags debug libssh"
+
+RDEPEND="
+	client? (
+		sys-libs/ncurses:=
+		sys-libs/readline:=
+	)
+	filecaps? (
+		acct-group/bird
+		acct-user/bird
+	)
+	libssh? ( net-libs/libssh:= )"
+BDEPEND="
+	app-alternatives/yacc
+	app-alternatives/lex
+	sys-devel/m4
+"
+
+FILECAPS=(
+	CAP_NET_ADMIN			usr/sbin/bird
+	CAP_NET_BIND_SERVICE	usr/sbin/bird
+	CAP_NET_RAW				usr/sbin/bird
+)
+
+src_prepare() {
+	default
+	eautoreconf
```

Comment:
I wonder if this is required?

Reply:
It doesn’t seem to been needed indeed, I’ll remove it for future ebuilds